.flowconfig

```diff
@@ -55,4 +55,4 @@ suppress_comment=\\(.\\|\n\\)*\\$FlowFixedInNextDeploy
 suppress_comment=\\(.\\|\n\\)*\\$FlowExpectedError
 
 [version]
-^0.63.0
+^0.65.0
```

Comment:
There is a more recent version of flow with far far better error descriptions, but currently it conflicts with some of the React Native flow types.